### PR TITLE
fix: "Add account" to "Connect Wallet"

### PR DESCRIPTION
**Pull request by AdarshRawat1**
Fixes _ https://github.com/getAlby/lightning-browser-extension/issues/2872
## "Accounts" (Manage accounts)
- Title changed to "Wallets"
- Button changed to "Connect a Wallet"
- removed outdated translations

**Current page** 
image

**Updated Page** 
image

## "Account settings" page

- Page title changes to "Wallet Settings"

**Current page**

image
image

**Updated Page** 

image
image

## "Settings" page
- Page title changes to "Extensions Settings"

**Current page**
image
image

**Updated Page** 
image
image

## "Manage accounts" --> "Manage Wallets"  "Add a new account" --> "Connect a Wallet"
**current** 
image
**Updated**
image

## Updated "Remove account"
image

# Modules renamed ( Important ) 
"options.account.account_settings"  ->  "options.account.wallet_settings"
"actions.remove_account"  - > "actions.disconnect_wallet"
"actions.add_account" -> "actions.connect_wallet"
### Link this PR to an issue [optional]

Fixes _  #2872

### Type of change

(Remove other not matching type)
- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [x] Self-review of changed code
- [x] Manual testing
- [x] Added automated tests where applicable
       - Updated Accounts test
       - Updated AccountMenu test
- [x] Update Docs & Guides
- For UI-related changes
- [x] Darkmode
- [x] Responsive layout

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
@reneaaron @bumi @stackingsaunter The PR is ready for review.

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -465,8 +465,8 @@
           }
         },
         "remove": {
-          "title": "Remove This Account",
-          "subtitle": "Removes all allowances, payment data and keys associated with this account.",
+          "title": "Disconnect This Wallet",
+          "subtitle": "All the linked allowances will be deleted. Please be certain.",
```
Comment (stackingsaunter) [inline src/i18n/locales/en/translation.json:469]: Change to: "Removes all allowances, payment data and keys associated with this wallet."

File: src/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -477,13 +477,13 @@
...
         "actions": {
-          "remove_account": "Remove account",
+          "disconnect_wallet": "Disconnect Wallet",
           "remove_secretkey": "Remove Master Key",
           "export": "Export"
         }
       },
       "actions": {
-        "add_account": "Add account"
+        "connect_wallet": "Connect a wallet"
```
Comment (stackingsaunter) [inline src/i18n/locales/en/translation.json:486]: Edit: Connect a **W**allet

**Review by stackingsaunter — COMMENTED** [review]
Two minor changes requested, besides it's good

**Comment by stackingsaunter** [comment]
@reneaaron I know you didn't like capitalisation. We are currently mixing both sentence and title case in buttons and dropdowns, there are 0 rules about this:

CleanShot 2023-12-20 at 02 44 35@2x

I just chose title case for this changes, I think it's slightly better but it's a matter of preference. Anyway, we can discuss it in another issue maybe and then change it all across all copy in the extension

**Pull request merged**